deploy/docker-compose.yml

```diff
@@ -1,53 +1,183 @@
-version: '3.1'
+version: "3.9"
 
 volumes:
   prometheus:
     driver: local
   alertmanager:
     driver: local
+  postgres:
+    driver: local
+  ipfs:
+    driver: local
+  geth:
+    driver: local
+  erigon:
+    driver: local
+  prysm:
+    driver: local
+  lighthouse:
+    driver: local
 
 services:
   oracle:
-    build:
-      context: ./../
-      dockerfile: Dockerfile
+    container_name: oracle
+    # build:
+    #   context: ./../
+    #   dockerfile: Dockerfile
+    image: europe-west4-docker.pkg.dev/stakewiselabs/public/oracle:feature-oracle-v2
     restart: always
     entrypoint: ["python"]
     command: ["oracle/oracle/main.py"]
-    env_file:
-      - .env
+    env_file: [".env"]
 
   # keeper:
+  #   container_name: keeper
   #   build:
```

Comment:
Can we use image for keeper here as well?

Reply:
Yes, added